examples/refsi/hal_refsi/include/device/program.lds

```diff
@@ -94,10 +94,9 @@ SECTIONS
   /*--------------------------------------------------------------------*/
   /* Local memory                                                       */
   /*--------------------------------------------------------------------*/
+  . = ALIGN(0x1000);
 
-  . = 0x10000000;
-
-  .local :
+  .local 0x10000000 :
```

Comment:
Can you explain what these two lines do? What I would expect is that if the second line here places `.local` at `0x10000000`, then the previous `. = ALIGN(0x1000);` is not necessary because `0x10000000` is always so aligned, but I am really not sure that my expectation is correct.

Reply:
Is that actually the effect? When I try it locally, changing GNU ld's default linker script to include
```
...
  . = ALIGN(0x80000);
  .text 0x100000  :
...
```
with a simple test program, then the presence or absence of `. = ALIGN(0x80000);` has no effect on the generated file, they are bitwise identical.

Reply:
Switched back to explicit memory regions, because specifying origin locations like this didn't actually work.